packages/wallets/src/features/cfd/screens/Jurisdiction/JurisdictionScreen.tsx

```diff
@@ -43,18 +43,20 @@ const JurisdictionScreen: FC<TJurisdictionScreenProps> = ({ selectedJurisdiction
                 ))}
             </div>
 
-            {selectedJurisdiction && selectedJurisdiction !== 'svg' && (
-                <div className='wallets-jurisdiction-screen__tnc'>
-                    Add Your Deriv MT5 Financial account under Deriv (V) Ltd, regulated by the Vanuatu Financial
-                    Services Commission.
-                    <div className='wallets-jurisdiction-screen__tnc-checkbox'>
-                        <input id='tnc-checkbox' type='checkbox' />
-                        <label htmlFor='tnc-checkbox' style={{ cursor: 'pointer' }}>
-                            <WalletText>I confirm and accept Deriv (V) Ltd&lsquo;s Terms and Conditions</WalletText>
-                        </label>
-                    </div>
-                </div>
-            )}
+            <div className='wallets-jurisdiction-screen__tnc'>
```

Comment:
This prevents the cards from shifting up and down when the TnC component is visible/hidden